FordLabs/PeopleMover
# 240 product link

## Issue
Resolves 240

## What was done
- turn product name into a link when a user inputs a URL on the product details modal

## How to test
1. create a product
2. add a URL to your product
3. see the pretty blue color on the name
4. click the link

## Accessiblity Criteria
### Testing

### Manual Checks
